python/send.py

```diff
@@ -526,6 +549,8 @@ def getJobInfo(argv):
             frun.write('cd $JOBDIR\n')
             if args.mergePileup:
                 frun.write('%s --inName %s\n'%(common_fccsw_command, all_inputs))
+            elif args.addPileupToSignal:
+                frun.write('%s --inName %s --mu %i --inPileupFileNames %s\n'%( common_fccsw_command, input_files[i], args.pileup, all_pileup_inputs))
```

Comment:
you already pass value of pileup in line 498

Reply:
true